INSTALL.md

```diff
@@ -49,20 +60,24 @@ sudo apt-get install -y libbpf-dev
 ### Install libbpf from source
 
 ```bash
-https_proxy="https://user:password@proxy.server:port" git clone https://github.com/libbpf/libbpf.git
+git clone https://github.com/libbpf/libbpf.git
 cd libbpf
-git checkout ...
-make -C src
-sudo make -C src install
+git checkout v0.6.1
```

Comment:
If we used 0.5.0 we can align with at least the Ubuntu 22.04 package manager's version. I suppose its ok to recommend the newer version; personally I just use the package manager's version.